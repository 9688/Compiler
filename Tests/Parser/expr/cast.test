(int)x;
(int)(char)x;
!(int)x;
(int)++x;