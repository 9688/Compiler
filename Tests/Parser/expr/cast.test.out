
list
     "CAST"
          int
          x
list
     "CAST"
          int
          "CAST"
               char
               x
list
     "!@"
          "CAST"
               int
               x
list
     "CAST"
          int
          "++@"
               x

